library/modules/form/controls/scss/input/floating.scss

```diff
@@ -14,21 +14,33 @@ $transition-speed: 0.2s;
 		width: 100%;
 		font-size: 1rem;
 		box-sizing: border-box;
-
+		&::-webkit-input-placeholder {
+			color: transparent;
```

Comment:
Please convert it into a loop to simplify the code, using an array or tuple for the placeholder variants.